doc/source/actions.rst

```diff
@@ -73,6 +73,33 @@ parameter:
 Once the backporting pull request is closed or merged, Mergify will
 automatically delete the backport head branch that it created.
 
+copy
+====
+
+The `copy` action just create a copy of the pull request as soon as the
+condition match.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1 2
+
+   * - Key Name
+     - Value Type
+     - Default
+     - Value Description
+   * - ``branches``
+     - array of string
+     - ``[]``
+     - The list of branches the pull request should be copied to.
+   * - ``regexes``
+     - array of string
+     - ``[]``
+     - The list of regexes to find branches the pull request should be copied to.
+
+Once the copy pull request is closed or merged, Mergify will
+automatically delete the copy head branch that it created.
```

Comment:
Shouldn't that be done by GitHub itself (they now have an option for it) or via `delete_head_branch`?

Reply:
I updated the default mergify rules.

Reply:
My point was more about not deleting it by default.
You can decide to delete it automatically with a rules, but you can't deactivate Mergify default rules.